ktorm-support-mysql/src/main/kotlin/me/liuwj/ktorm/support/mysql/Functions.kt

```diff
@@ -167,6 +167,22 @@ fun <T : Any> ColumnDeclaring<T>.ifNull(right: T?): FunctionExpression<T> {
     return this.ifNull(wrapArgument(right))
 }
 
+/**
+ * if function, translated to `if(condition, then, otherwise)`.
+ */
+fun <T : Any> IF(
+        condition: ColumnDeclaring<Boolean>,
+        then: ColumnDeclaring<T>,
+        otherwise: ColumnDeclaring<T>
```

Comment:
Please follow the existing code style, fix the indent.

Reply:
fixed